### PR TITLE
feat(azure-functions) update to new db & pdk

### PR DESCRIPTION
**Pull request by kikito**

### CODE REVIEW

File: kong/plugins/azure-functions/handler.lua [hunk 1]
```diff
@@ -97,15 +71,15 @@ function azure:access(config)
 
   local ok, err = client:connect(config.host, config.port)
   if not ok then
-    ngx_log(ngx.ERR, "[azure-functions] could not connect to Azure service: ", err)
-    return responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
+    kong.log.err("could not connect to Azure service: ", err)
+    return kong.response.exit(500, { msg = "An unexpected error ocurred" })
```
Comment (bungle) [inline kong/plugins/azure-functions/handler.lua]: Why is it `msg`, shouldn't it be `message`?

File: kong/plugins/azure-functions/handler.lua [hunk 2]
```diff
@@ -97,15 +71,15 @@ function azure:access(config)
...
   end
 
   if config.https then
     local ok, err = client:ssl_handshake(false, config.host, config.https_verify)
     if not ok then
-      ngx_log(ngx.ERR, "[azure-functions] could not perform SSL handshake : ", err)
-      return responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
+      kong.log.err("could not perform SSL handshake : ", err)
+      return kong.response.exit(500, { msg = "An unexpected error ocurred" })
```
Comment (bungle) [inline kong/plugins/azure-functions/handler.lua]: Ditto.

File: kong/plugins/azure-functions/handler.lua [hunk 3]
```diff
@@ -144,7 +118,8 @@ function azure:access(config)
   }
 
   if not res then
-    return responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
+    kong.log.err(err)
+    return kong.response.exit(500, { msg = "An unexpected error occurred" })
```
Comment (bungle) [inline kong/plugins/azure-functions/handler.lua]: Ditto.

**Review by bungle — CHANGES_REQUESTED** [review]
Soem changes requested.

File: kong/plugins/azure-functions/handler.lua (hunk 1 above)
Comment (kikito) [reply kong/plugins/azure-functions/handler.lua]: Lack of sleep does this. xD

**Review by bungle — APPROVED** [review]
LGTM

**Pull request merged**